odk1-src/briefcase-using.rst

```diff
@@ -204,9 +217,25 @@ To get help about the command line operation:
 Pulling form data from Aggregate
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: console
+- CLI flag: `-plla` or `--pull_aggregate`
+- Usage:
+
+  .. code-block:: console
+
+      $ java -jar {path/to/briefcase-jar-file} --pull_aggregate --form_id {form-id} --storage_directory {path/to/briefcase-storage-location} --aggregate_url {aggregate-url} --odk_username {username} --odk_password {password}
 
-  $ java -jar {path/to/briefcase-jar-file} --pull_aggregate --form_id {form-id} --storage_directory {path/to/briefcase-storage-location} --aggregate_url {aggregate-url} --odk_username {username} --odk_password {password}
+- Help section:
+
+  .. code-block:: console
+
+      Params for -plla operation:
+        -id,--form_id <arg>                 Form ID
+        -p,--odk_password <arg>             ODK Password
+        -sd,--storage_directory <arg>       Briefcase storage directory
+        -u,--odk_username <arg>             ODK Username
+        -url,--aggregate_url <arg>          Aggregate server URL
+      Optional params for -plla operation:
```

Comment:
line break above `Optional params` for readability.

Reply:
These blocks are verbatim copies of what you will get by running the `help` command in your console. There, required and optional params are grouped because it gives a better vertical rhythm since all blocks from all CLI operations are shown together.